server/services/messaging_service.go

```diff
@@ -303,7 +303,12 @@ func (s *messagingServerImpl) CreateBlurb(ctx context.Context, in *pb.CreateBlur
 	id := puid.Next()
 	name := fmt.Sprintf("%s/blurbs/%d", parent, id)
 	now := ptypes.TimestampNow()
-
+	switch legacyId := b.LegacyId.(type) {
+	case *pb.Blurb_LegacyRoomId:
+		name = fmt.Sprintf("%s/blurbs/%d.%s", parent, id, legacyId.LegacyRoomId)
+	case *pb.Blurb_LegacyUserId:
+		name = fmt.Sprintf("%s/blurbs/%d~%s", parent, id, legacyId.LegacyUserId)
```

Comment:
These format strings need to be updated to reflect the updated patterns.

Reply:
thank you! done.